IDENTIFICATION OF CANDIDATE KNEE OSTEOARTHRITIS BIOMARKER GENES CONSERVED BETWEEN HUMANS AND MICE FROM A CROSS-SPECIES SYSTEMS ANALYSIS


![image](https://github.com/Adrien1995/IDENTIFICATION-OF-CANDIDATE-KNEE-OSTEOARTHRITIS-BIOMARKER-GENES-CONSERVED-BETWEEN-HUMANS-AND-MICE-FR/assets/60327600/44913b08-90d6-4e30-ad4d-887e189c4626)


Abstract:


Following human patients longitudinally with osteoarthritis (OA) is not feasible because clinical observations are based on end-stage disease from operative specimens. This limitation leads to a dependency on mouse models that may not recapitulate all aspects of clinical OA progression. In this study, we leveraged transcriptomic profiles of osteoarthritic knee cartilage from mice and humans to perform a comprehensive cross-species cartilage co-expression network analysis. Gene expression datasets of OA mouse models were generated internally, and compared to gene expression datasets of human clinical samples that were publicly available. We established an understanding of relevant gene modules preserved between mice and humans. Furthermore, this analysis identified similarities and differences in disease mechanism among our noninvasive in-vivo loading model, DMM surgical model, and human clinical datasets of OA progression. We investigated the relevance of the identified cartilage hub genes as potential biomarkers in blood transcriptome data from clinical patients. We found that a gene module enriched for pre-hypertrophic chondrocytes was highly correlated to OA progression across both mouse models and human samples. Seven of the pre-hypertrophic chondrocyte markers, Col3a1, Col5a1, Tgfb1, Lrrc15, Col5a1, Postn, Tgfb1, Dkk3 and Mmp2, were identified as hub genes. Using the top 20 genes in the module, these markers were able to classify OA and healthy subjects in a blood human transcriptomic dataset with high sensitivity and specificity. Based on our findings, mouse models of load-induced OA can be used to study the disease mechanisms driven by chondrocytes populations linked to disease progression in clinical patients.

Structure

  1.code/ contains the code for cleaning, adjusting and normalizing individual datasets and performing cross-species transcriptome analyess
  
  2.raw_data/ contains individual-level study data for microarray, RNAseq, and single-cell RNAseq
  
  3.results/ contains figures and table results from the cross-species analyses
  
  4.working_data/ contains intermediary files

*available upon publication
